src/app/components/player/equipment/EquipmentPresets.tsx

```diff
@@ -42,6 +44,22 @@ const EquipmentPresets: React.FC = () => {
         };
         break;
       }
+      case EquipmentPreset.MAX_MAGE: {
+        newPlayer = {
+          equipment: {
+            head: findItemById(21018), // Ancestral hat
+            cape: findItemById(21791), // Imbued saradomin cape
+            neck: findItemById(12002), // Occult necklace
+            ammo: findItemById(22947), // Rada's blessing 4
+            body: findItemById(21021), // Ancestral robe top
+            legs: findItemById(21024), // Ancestral robe bottom
+            hands: findItemById(19544), // Tormented bracelet
+            feet: findItemById(13235), // Eternal boots
```

Comment:
Not sure if I should include mage boots. It's very common in raids to not bring mage boots because they are kind of poo.

Reply:
If we want to mirror what users are accustomed to we should look at what's used in the spreadsheet. I think this is fine though, since it's meant to be max gear.

Reply:
The Bitterkoekje sheet does use eternal boots so I guess I will leave it in